snuba/query/allocation_policies/__init__.py

```diff
@@ -741,6 +758,8 @@ def get_quota_allowance(
                 allowance = QuotaAllowance(True, self.max_threads, {})
             else:
                 allowance = self._get_quota_allowance(tenant_ids, query_id)
+        except InvalidTenantsForAllocationPolicy as e:
+            allowance = QuotaAllowance(False, 0, e.to_dict())  # type: ignore
```

Comment:
Why the #type: ignore here?

Reply:
Removed. The serializable exception conforms to the type but mypy is too dumb to realize it